### PR TITLE
Added screenshot to README

### PR DESCRIPTION
**Pull request by nishanth1232**
I have added a screenshot of the application as said in my google code-in task.

### CODE REVIEW

**Comment by nishanth1232** [comment]
Done.

File: README.md [hunk 1]
```diff
@@ -2,4 +2,5 @@ Konfetti
 ========
 Konfetti is a little screensaver which draws (semi-transparent) confetti on your desktop.
 
-Available on HaikuWare: http://haikuware.com/directory/view-details/utilities/screensavers/konfetti
+## Screenshot
```
Comment (humdingerb) [inline README.md]: I'd say it's obvious that what follows is a screenshot :)
Let's remove his heading and put a ":" at the end of the previous line.

**Comment by nishanth1232** [comment]
@humdingerb I didn't quite understand what you meant to say. Can you explain what edit's I have to make exactly?

**Comment by humdingerb** [comment]
I meant to remove the `## Screenshot` heading and put a ":" at the end of the description so it says, "Konfetti is a little screensaver which draws (semi-transparent) confetti on your desktop:"
Then follows the screenshot.

**Comment by nishanth1232** [comment]
Done.

**Pull request merged**